packages/wpcom.js/docs/site.md

```diff
@@ -303,3 +303,33 @@ site.statsVisits( function ( err, data ) {
 	// data visits response
 } );
 ```
+
+## Email Stats
+
+### Site#emailStatsAlltime([query, ]fn)
+
+Fetches the email stats for the whole period. This fetches the clients, devices & countries.
+
+It requires these params:
+
+- postId - the id of the post which email we are querying
+- statType - The type of stat we are working with. For example: 'opens' for Email Open stats
+- fn - a callback function to be executed (not required)
+
+It returns a request handler.
+
+### Site#emailStatsForPeriod(period, [query, ]fn)
```

Comment:
Same here, the signature is not right: `emailStatsForPeriod( postId, statType, query, fn )`